### ISSUE TITLE
[Enterprise] Issue within GridPanel. Function processKeyboardEvent should be improved.

### ISSUE THREAD

**Issue by rkhramiankou** [ag-grid/ag-grid#3088]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of ag-Grid'
[] support request => see 'Requesting Community Support'
```

**Providing a Reproducible Scenario**
https://github.com/ag-grid/ag-grid/blob/master/packages/ag-grid-community/src/ts/gridPanel/gridPanel.ts#L499
Condition to check !cellComp on the line 505 obviously should be placed right after cellComp value initialization;

I'm trying enterprise version before purchase to make sure all cases covered for our customer. After I've implemented some input controls inside the row group full width renderer console errors come out on keydown events. Seems like full width group row doesn't treat like cell, metadata with domDataKey is missed for it.

**Current behavior**
Console errors 'Cannot read property 'getRenderedRow' of null'

**Expected behavior**
No errors.

**Please tell us about your environment:**
Win 10

* **ag-Grid version:** X.X.X
ag-grid-enterprise": "^20.2.0"

* **Browser:**
Google Chrome Version 73.0.3683.86

* **Language:**
TypeScript 3.4.3

**Comment by gportela85** [comment]
fixed by https://github.com/ag-grid/ag-grid/commit/f011eb6e0dec84f3153de3add031308685b12856#diff-0151b9cfc42180be3e25ca8bbe7475d7

**Comment by GururajHShetty** [comment]
> fixed by [f011eb6#diff-0151b9cfc42180be3e25ca8bbe7475d7](https://github.com/ag-grid/ag-grid/commit/f011eb6e0dec84f3153de3add031308685b12856#diff-0151b9cfc42180be3e25ca8bbe7475d7)

is it officially released now?
